content/departments/engineering/engineering/admin-exp/devops/managed/index.md

```diff
@@ -164,6 +164,19 @@ Terraform is used to maintain all managed instances. You can find this configura
 
 All customer credentials, secrets, site configuration, app and user configuration—is stored in Postgres only (i.e. on the encrypted GCP disk). This allows customers to enter their access tokens, secrets, etc. directly into the app through the web UI without transferring them to us elsewhere.
 
+### Operations
+
+Managed Instances v1.1 documentation can be found [here](./v1.1/index.md)
+
+- [Creating a Managed Instance](./creation_process.md)
```

Comment:
we already have a ToC here https://github.com/sourcegraph/handbook/blob/main/content/departments/engineering/engineering/admin-exp/devops/managed/operations.md

can we just point user to the above page?

I am hoping we can keep this page cleaner and the target audience is people outside of our team

Reply:
or maybe we can crates a new operation folder and nest all processes under it

Reply:
oooh, great call @michaellzc. I didn't even see the operations hyperlink, what do you think about adding as it's own header and a link to `operations.md` instead

Reply:
I am fine with it.

Maybe we can also add them here so it's more visible. https://github.com/sourcegraph/handbook/blob/main/content/departments/engineering/engineering/admin-exp/devops/managed/index.md?plain=1#L9-L19